src/components/ModalDetails/ModalDetails.js

```diff
@@ -16,6 +16,12 @@ import { ArrowBackSharp } from "@material-ui/icons";
 import SelectBox from "../DetailsSelectBox/DetailsSelectBox";
 import MultiSelectCombobox from "../MultiSelectCombobox/MultiSelectCombobox";
 
+import {
+  FUTURE_DATE_ALLOWED_COMPONENTS,
+  getMaxFutureDate,
+  formatDateToInput
+} from "../../utils/constants/futureDateControls";
+
 // value in data => [0] api label, [1] our UI label, [2] value,[3], required or not for editing, [4] control form type
 const ModalDetails = ({
   modalData,
```

Comment:
The would be added here:
```suggestion
const ModalDetails = ({
  allowFutureDates,
  modalData,
```

Reply:
updated. see comments below.